config/hlint.yaml

```diff
@@ -10,3 +10,48 @@
 - ignore: { name: Use <|> }
 - ignore: { name: Use lambda-case }
 - ignore: { name: Use <$> }
+- modules:
```

Comment:
I would like to start with allowing nothing and then selectively adding things, but doing that is blocked by https://github.com/ndmitchell/hlint/pull/1221. It would be really tedious to do without `name: Rattletrap*`.